x-pack/plugins/observability_solution/observability_ai_assistant_app/server/rule_connector/index.ts

```diff
@@ -112,7 +116,7 @@ export function getObsAIAssistantConnectorType(
     },
     renderParameterTemplates,
     executor(options) {
-      return executor(options, initResources);
+      return executor(options, initResources, alertDetailsContextService);
```

Comment:
`initResources` make it less transparent what arguments this takes. I'd prefer if we just pass "base" arguments, instead of the "refined" arguments. So in this case it would look something like:

```suggestion
      return executor(options, core, plugins, request, alertDetailsContextService);
```